scripts/update-size.js

```diff
@@ -6,13 +6,16 @@ const sizeLimit = require('size-limit');
 const root = path.join(__dirname, '..');
 const readmePath = path.join(root, 'README.md');
 const readme = fs.readFileSync(readmePath, 'utf-8');
+const sizeConfig = require('../.size-limit');
 
-const pkg = require(path.join(root, 'package.json'));
-const file = path.join(root, pkg.browser[pkg.module]);
-
-sizeLimit(file, { ignore: ['react', 'prop-types'] }).then(bytes => {
+sizeLimit(sizeConfig[0].path, sizeConfig[0].ignore).then(bytes => {
```

Comment:
We only have a single item in sizeConfig array, so it would be better to unpack that first item right away when requiring, like this:
```js
const sizeConfig = require('../.size-limit')[0];
```
and use it across the file without needing to access first item all over the place,

from what I see the second argument to `sizeLimit` is the opts object, so you should actually pass whole sizeConfig there (already being an object, not an array, after u make the first requested change), because what needs to be passed there is
```js
{ ignore: ['prop-types'] } 
```
and you are passing now
```js
['prop-types']
```

That's also probably why the reported size in the README has increased so much.

Reply:
Updated sizeLimit config